Formula/pdnsd.rb

```diff
@@ -24,6 +24,43 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    Before starting "pdnsd" using "brew services" or manually, make sure that,
+    #{etc}/pdnsd.conf, #{var}/log/pdnsd.log and directory #{var}/cache/pdnsd
```

Comment:
This is going to break future upgrades when some of these files/directories can no longer be overwritten. Can we work on a better solution here?